vm/isolated/isolated.go

```diff
@@ -24,9 +24,11 @@ func init() {
 }
 
 type Config struct {
-	Targets      []string `json:"targets"`       // target machines: (hostname|ip)(:port)?
-	TargetDir    string   `json:"target_dir"`    // directory to copy/run on target
-	TargetReboot bool     `json:"target_reboot"` // reboot target on repair
+	Host         string   `json:"host"`           // host ip addr
+	Targets      []string `json:"targets"`        // target machines: (hostname|ip)(:port)?
+	TargetDir    string   `json:"target_dir"`     // directory to copy/run on target
+	TargetReboot bool     `json:"target_reboot"`  // reboot target on repair
+	USBDevNums   []string `json:"usb_device_num"` // usb device number
```

Comment:
Let's change the comment to `usb device numbers as in /sys/bus/usb/devices/`.

Reply:
done!